website/source/guides/packer-on-cicd/build-virtualbox-image.html.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: guides
+sidebar_current: guides-packer-on-cicd-build-virtualbox
+page_title: Build a VirtualBox Image with Packer in TeamCity
+---
+
+# Build a VirtualBox Image with Packer in TeamCity
+
+This guide walks through the process of building a VirtualBox image using
+Packer on a new TeamCity Agent. Before getting started you should have access
+to a TeamCity Server.
+
+The Packer VirtualBox builder requires access to VirtualBox, which needs to run
+on a bare-metal machine, as virtualization is generally not supported on cloud
+instances. This is also true for the [VMWare](/docs/builders/vmware.html) and
+the [QEMU](/docs/builders/qemu.html) Packer builders.
```

Comment:
A bit late on the ball here but both GCP and Azure support nested virtualization. I think this should be rephrased to be accurate.